### PR TITLE
Correct typo in 'prvide'

### PR DESCRIPTION
**Pull request by lukorir**
To correct typo in 'prvide'.

### CODE REVIEW

**Comment by jonocarroll** [comment]
Thank you! It looks like this worked. I like that the title of the pull request shows what you're fixing.

Typo fixes are a really helpful contribution, but you can make them even more helpful by running a spellcheck over the entire file or even the entire project. In RStudio there is a menu option 'Edit -> Check Spelling...' with a default shortcut of F7. I asked people to fix just one thing, so it's entirely fine to leave this one as is, but if you'd like to try out adding more fixes, feel free to fix one or two more typos and push to your branch again.

For this specific fix, I am going to suggest you also change the text to "provide _a_ safe space" so you can see what a maintainer suggestion looks like.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <!-- badges: start -->
 <!-- badges: end -->
 
-The goal of safespace is to prvide safe space for new developers to try out a pull request (particularly fo [Hacktoberfest 2024](https://hacktoberfest.com/participation/)) 
+The goal of safespace is to provide safe space for new developers to try out a pull request (particularly fo [Hacktoberfest 2024](https://hacktoberfest.com/participation/))
```
Comment (jonocarroll) [inline README.md:7]:
```suggestion
The goal of safespace is to provide a safe space for new developers to try out a pull request (particularly fo [Hacktoberfest 2024](https://hacktoberfest.com/participation/)) 
```

**Comment by jonocarroll** [comment]
Looks Good To Me (LGTM)!

**Comment by jonocarroll** [comment]
Congrats on a successful pull request!

**Comment by jonocarroll** [comment]
You're free to either delete your repository (since it will be identical to mine as you used your main branch as well) or keep it, but it won't stay in sync with any changes made here. Pushes to your main branch will no longer be reflected here, but the commits made here have been 'pulled in' to this repository and are now part of its history, so you don't need to worry about them.

Let me know if you have any other questions.

**Comment by lukorir** [comment]
Thank you very much for this initiative. It is of great help to me, trying
to learn about how to contribute using github, git, Rstudio.

You mention that I am "free to either delete your repository (since it will
be identical to mine as you used your main branch as well) or keep it, but
it won't stay in sync with any changes made here". What would have been a
better approach to using "my main branch", to avoid the need to delete my
repository, but continue contributing without having to fork it from yours
every time I make changes?
Regards
Luke

**Comment by jonocarroll** [comment]
Using your main branch is usually fine, but if your pull request doesn't get merged for some reason then it won't be in the same state as the original, so I try to make my changes in a new branch of my fork, and occasionally sync my fork's main branch with the original (GitHub now has a button for this).

It also means that you retain a forked copy of the original that you haven't changed, in case you want to make some local comparisons.

If you create a pull request via the web editor it creates a 'patch-1' (or something similar) branch in your fork.

Does that make sense?

**Comment by jonocarroll** [comment]
In the git pane in RStudio there's a 'new branch' button that can be helpful.

If it helps, #2 has a hint about what's immediately broken in terms of the package.

**Pull request merged**